lib/gelf/logger.rb

```diff
@@ -5,7 +5,7 @@ module LoggerCompatibility
     attr_accessor :formatter
 
     # Use it like Logger#add... or better not to use at all.
-    def add(level, message = nil, progname = nil)
+    def add(level, message = nil, progname = nil, &block)
```

Comment:
Sorry for my late answer.
I'm not sure to get you well. Do you prefer to remove the `&block` param, or something else? Actually I just saw that Ruby's `Logger#add` removed it also at https://github.com/ruby/ruby/commit/460e9d8d83b22efad02f69925492195124f52a86.